andela/blade-runner-cfh
# Chore/153741751/configure hound ci

#### What does this PR do?

Add hound-ci style checking to project

#### Description of Task to be completed?

Add style checking for hound-ci on every pull request

#### How should this be manually tested?

- Create a pull request
- Notice hound ci style checking running on pull requests, and creating pull request comments

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

153741751

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
